src/app/ui/editor/standby_state.cpp

```diff
@@ -518,10 +531,6 @@ bool StandbyState::onSetCursor(Editor* editor, const gfx::Point& mouseScreenPos)
   if (editor->canDraw()) {
     editor->showBrushPreview(mouseScreenPos);
   }
-  // Forbidden
-  else {
-    editor->showMouseCursor(kForbiddenCursor);
```

Comment:
Are we sure that this will never happen again? Isn't needed for reference layers maybe?